content/posts/2019-11-07-ln-split-tickets-decred-blog.md

```diff
@@ -0,0 +1,159 @@
+
+As usual, the tricky bits are the nitty gritty details of just _how_ to orchestrate purchasing the ticket and then disbursing the stake rewards after a vote such that all participants are assured their funds are recoverable in the worst case scenario of a breakdown of communication.
+
+The [on-chain version](https://matheusd.com/post/ticket-splitting-challenges/) of split tickets naturally solves this problem by relying on the consensus rules that enforce the returns of the funds of the ticket to each original financer. Off-chain multi-owner tickets however, cannot rely on that given the UTXOs are generally locked into two-party contracts. 
+
+{{< legendfigure src="/images/posts/ln-splits-01-onchainsplit.png" alt="Typical on-chain split" title="Typical on-chain split ticket with 6 participants. Notice the return addresses are guaranteed by consensus rules so funds are completely safe after a vote or revocation." >}}
+
+For example, in the Lightning Network the channel balance is defined by the UTXO which locks funds in the multisig 2-of-2 script of the channel's parties. While it would be possible to use that UTXO to directly fund a ticket, that's still an on-chain operation and ends up having the same scalability problems as the current on-chain solution. Besides that, users might not want to commit all their available channel funds into tickets or only one of the parties might be interested in doing so.
+
+Therefore a viable solution for off-chain multi-owner tickets needs to run independently of any and all outstanding channels of the network. It needs to be able to aggregate funds from up to thousands of users into a single output that is used to fund a ticket and then allow these same users to redeem the stake rewards, potentially months after their initial coordinated purchase and accounting for all possible failure modes.
+
+After quite of bit of design work we're finally at a stage where we can show the outline of the solution for multi-owner split tickets over LN!
+
+It involves some interesting new smart contract constructions, new consensus enforced opcodes and leads naturally to several changes to the staking system.
+
+This will be a big undertaking which will require touching the entire stack of Decred software. It needs to be more carefully vetted by the larger stakeholder community and is something of a longer term project that will take months to get moving, with a successful payoff only being realized in the future where a much larger audience wants to participate in our on and off-chain decision making processes.
+
+But we do feel it's a worthwhile program to get going. And as will be presented in the next few sections, even if this ends up not being _the_ ultimate solution to multi-owner tickets, the byproducts of this work - the new constructions, opcodes and staking changes - are valuable contributions that enable new use cases for the coin.
```

Comment:
```suggestion
But we do feel it's a worthwhile program to get going. As will be presented in the next few sections, even if this ends up not being _the_ ultimate solution to multi-owner tickets, the byproducts of this work - the new constructions, opcodes and staking changes - are valuable contributions that enable new use cases for the coin.
```

Reply:
In the UK, we would use `programme` in this context.

> programme
> /ˈprəʊɡram/
> noun
> 1. a set of related measures or activities with a particular long-term aim.
> "an extensive programme of reforms"

However, as most of the Decred writing is American-ised, we should probably keep it as-is. Just highlighting something I find interesting :slightly_smiling_face: